test/gurka/test_match.cc

```diff
@@ -11,30 +11,48 @@ TEST(Standalone, BasicMatch) {
 
   const std::string ascii_map = R"(
       1
-    A---2B-3-4C
-              |
-              |5
-              D
+    A---2B-3-45C
+               |
+               |6
+               D
          )";
 
   const gurka::ways ways = {{"AB", {{"highway", "primary"}}},
                             {"BC", {{"highway", "primary"}}},
                             {"CD", {{"highway", "primary"}}}};
 
-  const double gridsize = 10;
+  const double gridsize = 9;
```

Comment:
9 meters ensures that node 5 will be set "interpolated" (the default `interpolation_distance` is 10)